homeassistant/components/nibe_heatpump/__init__.py

```diff
@@ -295,12 +295,12 @@ def _get_coils() -> Iterable[Coil]:
 
         return result
 
-    async def async_shutdown(self):
```

Comment:
This change should get its own PR.

Reply:
Agreed - now that `async_shutdown` is async it can be done in a follow-up PR.